kuma/payments/jinja2/payments/thank_you.html

```diff
@@ -71,8 +74,62 @@ <h2 class="highlight highlight-spanned">
+                    <div class="faq-feedback">
+                        <div class="sub-section">
+                            <h3>{{_('Managing your payments')}}</h3>
+                            <p>
+                                {% trans url="mailto:" + settings.CONTRIBUTION_SUPPORT_EMAIL + "?Subject=Manage%20monthly%20payment",
+                                         email=settings.CONTRIBUTION_SUPPORT_EMAIL %}
+                                    If you would like to manage your monthly payment,
+                                    such as changing your card account details or the
+                                    amount you pay, please contact
+                                    <a href="{{ url }}">{{ email }}</a>.
+                                {% endtrans %}
+                            </p>
+                        </div>
+                        <div class="sub-section">
+                            <h3>{{_('Cancelling your payments')}}</h3>
+                            <p>
+                                {% trans url="mailto:" + settings.CONTRIBUTION_SUPPORT_EMAIL + "?Subject=Cancel%20monthly%20payment",
+                                    email=settings.CONTRIBUTION_SUPPORT_EMAIL %}
+                                    If you would like to cancel your monthly payment,
+                                    you are free to do so at any point. Please contact
+                                    <a href="{{ url }}">{{ email }}</a> to cancel.
+                                {% endtrans %}
+                            </p>
+                        </div>
+                        <div class="sub-section">
+                            <h3>{{_('Payment terms')}}</h3>
+                            <p>
+                                {% trans url="https://stripe.com/us/checkout/legal" %}
+                                    Please read our
+                                    <a href="{{ url }}">payment terms</a>
+                                    for more information.
+                                {% endtrans %}
+                            </p>
+                        </div>
+                        <div>
+                            <h3>{{_('FAQ')}}</h3>
+                            <p>
+                                {% trans url=url('payments') %}
+                                    To find out more about why MDN is raising money
+                                    through the one time and monthly payment programme,
+                                    <a href="{{ url }}">please visit our FAQ</a>.
+                                {% endtrans %}
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+        {% endif %}
         <section class="section">
-            <img src="{{ static('img/hero-dino-blank.png') }}" class="backdrop-image" alt="" role="presentation" />
+            <!-- TODO: replace with waffle flag/varible -->
```

Comment:
Nit: If you use Jinja2 comments, they don't appear in the rendered HTML

```suggestion
            {# TODO: replace with waffle flag/variable #}
```